src/Avalonia.Controls/Automation/Peers/AutomationPeer.cs

```diff
@@ -250,6 +255,7 @@ protected virtual string GetLocalizedControlTypeCore()
         protected abstract string GetClassNameCore();
         protected abstract AutomationPeer? GetLabeledByCore();
         protected abstract string? GetNameCore();
+        protected abstract string? GetHelpTextCore();
```

Comment:
Unfortunately, adding an `abstract` method to a public type is a major breaking change, which we can't accept.
The new method will have to be `virtual` for now, and return `null` by default.

Reply:
Adding to V12 cleanup list